compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/reference/ClassReference.kt

```diff
@@ -118,6 +125,16 @@ public sealed class ClassReference {
       clazz.annotationEntries.map { it.toAnnotationReference(module) }
     }
 
+    private val directSuperClassReferences: Sequence<ClassReference> by lazy(NONE) {
+      if (clazz is KtEnumEntry) {
+        emptySequence()
```

Comment:
Hm, I know this if-check is coming from the original utility. But I believe it's wrong. An enum can implement other interfaces.